documentation/modules/ref-operator-cluster.adoc

```diff
@@ -121,6 +118,31 @@ The DNS domain name is added to the Kafka broker certificates used for hostname
 +
 If you are using a different DNS domain name suffix in your cluster, change the `KUBERNETES_SERVICE_DNS_DOMAIN` environment variable from the default to the one you are using in order to establish a connection with the Kafka brokers.
 
+== Logging configuration
```

Comment:
@PaulRMellor @laidan6000 
Could you please check these changes?

Reply:
@stanlyDoge :+1: